client/components/feature-modules/blocks/components/modals/type-picker-modal.tsx

```
@@ -0,0 +1,234 @@
+                            <div className="flex-1">
+                                <div className="flex items-center gap-2">
+                                    {Icon && <Icon className="h-4 w-4 text-muted-foreground" />}
+                                    <span className="font-medium">{label}</span>
+                                </div>
+                                {description && (
+                                    <p className="text-sm text-muted-foreground mt-1">
+                                        {description}
+                                    </p>
+                                )}
+                            </div>
+                        </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)